x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/AbstractSqlQueryRequest.java

```diff
@@ -514,49 +510,4 @@ public int hashCode() {
             runtimeMappings
         );
     }
-
-    // This is needed just to test round-trip compatibility with proto.SqlQueryRequest
-    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
```

Comment:
The main advantage of this commit is removing this duplicated code from the action classes.